lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -74,7 +81,11 @@
                 },
 
                 render: function() {
-                    this.$el.html(_.template(editTeamTemplate)({primaryButtonTitle: this.primaryButtonTitle}));
+                    this.$el.html(_.template(editTeamTemplate)({
+                        primaryButtonTitle: this.primaryButtonTitle,
+                        isNew: this.action === 'create',
```

Comment:
Passing in the `action` instead of a boolean will be nicer since it describes the purpose of the form.